students/17th/team1/gyuboshim/westagram/user/views.py

```diff
@@ -0,0 +1,76 @@
+from django.shortcuts   import render
+from django.views       import View
+from django.http        import HttpResponse, JsonResponse
+from django.db.models   import Q
+
+from .models            import User
+
+import json
+
+PASSWORD_LEN_LIMIT = 8
+
+class UserView(View):
+    def post(self, request):
+        try:
+            data            =   json.loads(request.body)
+            user_data       =   data['user']
+            phone_number    =   user_data['phone_number']
+            email_adress    =   user_data['email_adress']
+            name            =   user_data['name']
+            nickname        =   user_data['nickname']
+            password        =   user_data['password']
+
+            if (email_adress or phone_number) and password:
+                
+                if User.objects.filter(email_adress = email_adress).exists():
+                    return JsonResponse({"message":"EMAIL_ALREADY_EXISTS"}, status = 409)
+                
+                if User.objects.filter(phone_number = phone_number).exists():
+                    return JsonResponse({"message":"PHONENUMBER_ALREADY_EXISTS"}, status = 409)
+
+                if '@' or '.' not in email_adress:
+                    return JsonResponse({"message":"@_OR_._DOES_NOT_EXIST"}, status = 400)
+
+                if len(password) < PASSWORD_LEN_LIMIT:
+                    return JsonResponse({"message": "PASSWORD_HAS_TO_BE_AT_LEAST_8-DIGIT"}, status = 400)
```

Comment:
```suggestion
                    return JsonResponse({"message": "SHORT_PASSWORD"}, status = 400)
```
이정도면 될것 같습니다. 바로 메세지로서의 에러코드 이기 때문에 너무 설명적이지 않도록 해주세요.